### PR TITLE
feat(docs): Add quick start

### LINKED ISSUE #4: [document request] Add Quick Start
/priority p0

### PR DESCRIPTION
**Pull request by gaocegege**
**What this PR does / why we need it**:

Add quick start

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #4 

Reference to #

**Special notes for your reviewer**:

/cc @ddysher 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: docs/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+size:    6.1 KiB
+Status: Downloaded newer model for localhost:5000/ormb/pmml:v1
+```
+
+## Export the model to the directory
+
+We can export the model from the local cache to the target directory.
+
+```bash
+$ ormb pull localhost:5000/ormb/pmml:v1
```
Comment (ddysher) [inline docs/tutorial.md:96]: should this be `export`?

**Comment by gaocegege** [comment]
Comments are addressed, PTAL

**Pull request merged**